src/config-hardware/fix-audio.md

```diff
@@ -0,0 +1,77 @@
+# Useful Audio Commands
+
+Use Terminal commands to restart daemons and drivers that control audio streams.
+
+---
+
+## ALSA Commands
+
+ALSA commands can be used to view audio-related diagnostic information and to reset audio drivers.
+
+View all devices detected by ALSA:
+
+```bash
+aplay -l
+```
+
+View all recording devices detected by ALSA:
+
+```bash
+arecord -l
+```
+
+Reload sound driver modules: 
+
+```bash 
+sudo alsa force-reload
+```
+
+The `alsa-info` command gathers a number of outputs, including some of the above-listed outputs, and packages them for convenient sharing.
+
+```bash
+alsa-info
+```
+
+## Additional Audio Commands
+
+### List Detected Sound Cards
+
+If ALSA doesn't list a sound card, it may not be physically detected by the system at all. If the Linux kernel sees a sound card, it will show up in your `lspci` output. This command will list every sound card your system detects, and show the driver being used for each one:
+
+```bash
+lspci -v | grep -A6 Audio
+```
+
+### Check PipeWire's Status
+
+This command will check the status of PipeWire and show any errors if automatic restarts raised any errors:
+
+```bash
+systemctl --user status pipewire
+```
+
+### Reset PipeWire to Defaults
+
+Clearing the current settings for Pipewire  may allow the defaults to be used again. To revert to defaults and clear any current saved settings run the following commands:
```

Comment:
> for Pipewire

`PipeWire` should have the W capitalized. Also, there is an extra space after that word.

> may allow the defaults to be used again.

What does "may allow" mean? In what situation would that not be the case?

> To revert to defaults and clear any current saved settings run the following commands:

There should be a comma after the word `settings`. `current saved` should probably be `currently saved`.

Also, this would not reset any system-wide settings that were altered (e.g. in `/etc`.)

Reply:
This section is taken from a support article and I should have paid better attention to the written content. I've added your corrections and clarified that this command resets the current user's local settings.